tests/test_comparator.rs

```diff
@@ -39,19 +39,15 @@ pub fn write_to_db_with_comparator(compare_fn: Box<CompareFn>) -> Vec<String> {
 
 #[test]
 /// First verify that using a function as a comparator works as expected
-/// This should verify backwards compatablity
+/// This should verify backwards compatibility
 /// Then run a test with a clojure where an x-variable is passed
 /// Keep in mind that this variable must be moved to the clojure
 /// Then run a test with a reverse sorting clojure and make sure the order is reverted
 fn test_comparator() {
     let local_compare = move |one: &[u8], two: &[u8]| one.cmp(two);
 
-    let x = 0;
     let local_compare_reverse = move |one: &[u8], two: &[u8]| {
-        println!(
```

Comment:
These changes of `local_compare_reverse` is not intended. Will be reverted in next commit.